lib/stack_master/cli.rb

```diff
@@ -178,7 +178,7 @@ def execute!
           options.defaults config: default_config_file
           say "Invalid arguments. stack_master tidy" and return unless args.size == 0
           config = load_config(options.config)
-          StackMaster::Commands::Tidy.perform(config)
+          StackMaster::Commands::Tidy.perform(config, options)
```

Comment:
Shouldn't the signature be `config, nil, option` ?

Reply:
You're right.

Reply:
Fixed in 6256955